src/core/Navigation/WizardNavigation/WizardNavigation/WizardNavigation.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { forwardRef, ReactNode, useState } from 'react';
+import { HtmlButton, HtmlDiv, HtmlNav, HtmlUl } from '../../../../reset';
+import styled from 'styled-components';
+import { SuomifiThemeConsumer, SuomifiThemeProp } from '../../../theme';
+import { baseStyles } from './WizardNavigation.baseStyles';
+import classnames from 'classnames';
+import { Icon } from '../../../Icon/Icon';
+import { getConditionalAriaProp } from '../../../../utils/aria';
+
+export interface WizardNavigationProps {
+  /** Use the `<WizardNavigationItem>` components as children */
+  children: ReactNode;
+  /** Name for the navigation element. Don't use the word "navigation" since it will be read by screen reader regardless. */
+  'aria-label': string;
+  /** HTML nav element will receive this id */
+  id?: string;
+  /**
+   * Normal or small screen variant
+   * @default normal
+   */
+  variant?: 'default' | 'smallScreen';
+  /**
+   * Whether the menu is initially expanded. Only applies to smallScreen variant.
+   * @default true
+   */
+  initiallyExpanded?: boolean;
+  /** Navigation component heading text */
+  heading: string;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Ref is forwarded to nav element. Alternative for React `ref` attribute. */
+  forwardedRef?: React.RefObject<HTMLDivElement>;
```

Comment:
Also in here the type could be `React.Ref<HTMLElement>` if it's changed in HtmlNav

Reply:
Fixed this!